### PR TITLE
State the minimum version of PHPUnit in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SpacePossum**
run into the same as this; https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/899

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
  * Install dependencies: `composer install`.
- * Create branch, e.g. feature-foo or bugfix-bar.
+ * Create branch, e.g. `feature-foo` or `bugfix-bar`.
  * Make changes.
  * If you are adding functionality or fixing a bug - add a test!
  * Fix project itself: `php php-cs-fixer fix`.
- * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify README.rst manually!
- * Check if tests pass: `phpunit`.
+ * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify `README.rst` manually!
+ * Check if tests pass: [`PHPUnit`](https://phpunit.de/manual/current/en/installation.html) (>=4.0)
```
Comment (keradus) [inline CONTRIBUTING.md]: before it was a command for CLI, such as `composer install`, `php php-cs-fixer fix` etc. Not it is not.

Comment (SpacePossum) [inline CONTRIBUTING.md]: Fair point, I've updated so it reflects a command again.

**Comment by keradus** [comment]
:+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
  * Install dependencies: `composer install`.
- * Create branch, e.g. feature-foo or bugfix-bar.
+ * Create branch, e.g. `feature-foo` or `bugfix-bar`.
  * Make changes.
  * If you are adding functionality or fixing a bug - add a test!
  * Fix project itself: `php php-cs-fixer fix`.
- * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify README.rst manually!
- * Check if tests pass: `phpunit`.
+ * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify `README.rst` manually!
+ * Check if tests pass: `phpunit` [(>=4.0)](https://phpunit.de/manual/current/en/installation.html)
```
Comment (GrahamCampbell) [inline CONTRIBUTING.md]: Maybe we should just say phpunit 4 is required. It is still possible for later versions of phpunit not to work.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
 # Contributions are welcome!
 
-## Quick guide:
+## Quick guide
 
  * Fork the repo.
+ * Checkout the branch you want to make changes on.
  * Install dependencies: `composer install`.
- * Create branch, e.g. feature-foo or bugfix-bar.
+ * Create branch, e.g. `feature-foo` or `bugfix-bar`.
```
Comment (keradus) [inline CONTRIBUTING.md]: Why did you quote this?

Comment (keradus) [inline CONTRIBUTING.md]: actually why not

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by SpacePossum** [comment]
:+1:  thank you @keradus

**Comment by GrahamCampbell** [comment]
@keradus When is this 1.4 release coming?

**Comment by keradus** [comment]
I have no rights to release it.

**Comment by GrahamCampbell** [comment]
Yeh. We really need to get that sorted out.

**Pull request closed without merging**